src/server-extension/resolvers/series.ts

```diff
@@ -11,6 +11,7 @@ enum OrderBy {
   total = 'total',
   average_price = 'average_price',
   floor_price = 'floor_price',
+  highest_sale = 'highest_sale'
```

Comment:
```suggestion
  highest_sale = 'highest_sale',
```
missing comma

Reply:
Since we add a new column, I am thinking that we should update DB migrations?
Don't know if I understand right.

> do you think it's better to remove burned nft in another PR?

Yes, cause I also want to do some refactoring, and extract those same queries into one file.